tests/functional/settings.js

```diff
@@ -82,39 +81,26 @@ define([
 
     'sign in to desktop context, go to settings, no way to sign out': function () {
       var self = this;
-      return this.remote
-        .get(require.toUrl(SIGNIN_URL + '?context=' + Constants.FX_DESKTOP_V1_CONTEXT))
-        .setFindTimeout(intern.config.pageLoadTimeout)
-        .execute(listenForFxaCommands)
+      var url = SIGNIN_URL + '?context=' + Constants.FX_DESKTOP_V1_CONTEXT + '&service=sync';
 
-        .findByCssSelector('form input.email')
-          .click()
-          .type(email)
-        .end()
-
-        .findByCssSelector('form input.password')
-          .click()
-          .type(FIRST_PASSWORD)
-        .end()
+      return FunctionalHelpers.openPage(self, url, '#fxa-signin-header')
+        .execute(listenForFxaCommands)
 
-        .findByCssSelector('button[type="submit"]')
-          .click()
-        .end()
 
         .then(function () {
-          return testIsBrowserNotifiedOfLogin(self, email);
```

Comment:
This PR uncovered the fact that this test was incorrect, and because of lucky timing, not failing. The account used to sign in was verified, yet `testIsBrowserNotifiedOfLogin` should have been failing because by default, it checks if the account is unverified.